opentok-angular.js

```diff
@@ -311,3 +293,94 @@ ng.module('opentok', [])
       };
     }
   ]);
+
+  const captionSubscriberTracker = (OTSession, $rootScope) => {
+    const MAX_SUBS_ON_SCREEN = 4;
```

Comment:
why 4?

Reply:
It's an arbitrary number. In one of the meetings I believe Anand or Binoy had suggested it, but I'm happy to change it. We don't want it too high where the entire screen gets filled up with text, but maybe 5 or 6 would be better? I don't have any strong opinions.

Reply:
cool; thanks, man 🙏